### PR TITLE
Nrpe command template

### PR DESCRIPTION
**Pull request by bashrc666**
Hi,

I made some change about your ansible_nrpe template by making vars dict to generate this one .
I've update the doc.

Let me know if i've done wrong.

Best regards

Greg

### CODE REVIEW

**Comment by jloh** [comment]
Thanks for much for this! Sorry its taken be a while to get back to you on this, I'm hoping to have a closer look next week.

File: README.md [hunk 1]
```diff
@@ -96,12 +96,21 @@ Example Playbook
...
      nagios_nrpe_command:
+<<<<<<< HEAD
+      oracle_tnsping:
+        script: check_oracle_health
+        option: --mode tnsping
+      oracle_connection-time:
+        script: check_oracle_health
+        option: --mode connection-time
+=======
        oracle_tnsping:
```
Comment (jloh) [inline README.md]: @bashrc666 looks like you've got a merge conflict here?

File: README.md [hunk 2]
```diff
@@ -96,12 +96,21 @@ Example Playbook
...
          script: check_oracle_health
          option: --mode tnsping
        oracle_connection-time:
          script: check_oracle_health
          option: --mode connection-time
+>>>>>>> 281b6b540002e8fc380593317a02129558d21b57
```
Comment (jloh) [inline README.md]: And here as well

**Comment by bashrc666** [comment]
Sorry for that i've made some change again.
I've change the parameter to a var for dont_blame_nrpe.

**Comment by jloh** [comment]
Thanks again!

**Pull request merged**